libpod/util.go

```diff
@@ -149,6 +149,14 @@ func queryPackageVersion(cmdArg ...string) string {
 		cmd := exec.Command(cmdArg[0], cmdArg[1:]...)
 		if outp, err := cmd.Output(); err == nil {
 			output = string(outp)
+			if cmdArg[0] == "/usr/bin/dpkg" {
+				r := strings.Split(output, ": ")
+				queryFormat := `${Package}_${Version}_${Architecture}`
+				cmd = exec.Command("/usr/bin/dpkg-query", "-f", queryFormat, "-W", r[0])
+				if outp, err := cmd.Output(); err == nil {
+					output = string(outp)
+				}
+			}
```

Comment:
Instead of making a special case here, `/usr/bin/dpkg-query", "-f", queryFormat, "-W"` should just replace `"/usr/bin/dpkg", "-S"` on line 172 or am I missing something.

Reply:
One command takes a path, one command takes a package ?

Reply:
I think you can also use `dpkg-query` with `-S`

Reply:
I was not able to make it work, but maybe you have a different version of `dpkg` ?

It works, one at a time:

```console
$ dpkg-query -S /usr/bin/runc
containerd.io: /usr/bin/runc
$ dpkg-query -W containerd.io
containerd.io	1.4.12-1
```

But not as a one-liner:

```console
$ dpkg-query -W -S /usr/bin/runc
dpkg-query: error: conflicting actions -S (--search) and -W (--show)
```

Reply:
I don't have access to a debian system right now. I just looked at the man page and it doesn't mention the show and search conflict so I assumed it will work.